test.sh

```diff
@@ -10,9 +10,6 @@ test_err() {
 }
 
 test_js() {
-    if [[ ! -x ./node_modules/karma/bin/karma || ! -x ./node_modules/jshint/bin/jshint ]]; then
-        npm install
-    fi
     npm run lint -- "${jsfiles[@]}"
```

Comment:
This needs to be:

```sh
jshint --config=etc/jshint.conf "${jsfiles[@]}"
``` 

The global install doesn't not (thankfully) install the scripts:

```js
  "scripts": {
    "lint": "jshint --config=etc/jshint.conf",
    "test": "karma start etc/karma-conf.js"
  },
```

These should be removed from package.json, too, but at a later date.

Reply:
0d0cbd9727a002b1ea964084a02b7d51b8ab1f25